src/server/page_server.hpp

```diff
@@ -46,11 +46,13 @@ class page_server
     ~page_server ();
 
     void set_active_tran_server_connection (cubcomm::channel &&chn);
-    void start ();
-    void disconnect_active_tran_server ();
+    void set_passive_tran_server_connection (cubcomm::channel &&chn);
+    void disconnect_tran_server (void *conn);
```

Comment:
Be specific with the type of argument. It is not a void pointer, it is a connection_handler. Compiler will complain, so make sure that connection_handler is forwarded.
```suggestion
    void disconnect_tran_server (connection_handler *conn);
```